generator/config/expression/case.yaml

```diff
@@ -3,7 +3,8 @@ name: $case
 link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/switch/'
 type:
     - switchBranch
-encode: flat_object
+encode: object
+noName: true
```

Comment:
Per #1540, should this be `mergeObject`?

Reply:
This is not the same property. Here, we don't wrap the properties into an object with the operator name as single property. `{ $operator: { ... properties }}` vs `{ ... properties }`. Maybe `wrapObject: false`